tools/tsp-client/test/commands.spec.ts

```diff
@@ -238,4 +240,44 @@ describe.sequential("Verify commands", () => {
       assert.fail("Failed to init. Error: " + error);
     }
   });
+
+  it("Generate config files", async () => {
+    try {
+      const args = {
+        "package-json": joinPaths(cwd(), "./test/package.json"),
+      };
+      repoRoot = await getRepoRoot(cwd());
+      await generateConfigFilesCommand(args);
+      assert.isTrue(await doesFileExist(joinPaths(repoRoot, "eng", "emitter-package.json")));
+      const emitterJson = JSON.parse(
+        await readFile(joinPaths(repoRoot, "eng", "emitter-package.json"), "utf8"),
+      );
+      assert.equal(emitterJson["dependencies"]["@azure-tools/typespec-python"], "0.37.3");
```

Comment:
I would add a devDependency assertion:

```suggestion
      assert.equal(emitterJson["dependencies"]["@azure-tools/typespec-python"], "0.37.3");
      assert.equal(emitterJson["devDependencies"]["@azure-tools/typespec-azure-core"], "~0.49.0");
```

Reply:
added